hack/labeled-squashfs.sh

```diff
@@ -0,0 +1,6 @@
+#!/toolchain/bin/bash
+
+set -e
+# set SELinux labels for files according to file_contexts supplied
+/toolchain/sbin/setfiles -r $1 -F -vv $3 $1
```

Comment:
would this adjust the modified time that we want to preserve? do even need this to be a shell script? can we instead use `RUN <<END` ?